RealmSwift-swift3.0-dev/List.swift

```diff
@@ -31,7 +31,7 @@ public class ListBase: RLMListBase {
         return descriptionWithMaxDepth(depth: RLMDescriptionMaxDepth)
     }
 
-    @objc private func descriptionWithMaxDepth(depth: UInt) -> String {
+    @objc(descriptionWithMaxDepth:) private func descriptionWithMaxDepth(depth: UInt) -> String {
```

Comment:
Changing the Swift function signature to `@objc private func descriptionWithMaxDepth(_ depth: UInt) -> String` seems to work. (I had to explicitly omit the first parameter.)